test/integration/virtual-rules/empty-table-header.js

```diff
@@ -0,0 +1,142 @@
+describe('empty-table-header virtual-rule', function () {
```

Comment:
We'll want a test that should return incomplete if the `th` element does not have the `children` set to an empty array. Similar to [button-name](https://github.com/dequelabs/axe-core/blob/develop/test/integration/virtual-rules/button-name.js#L116-L125)